### ISSUE TITLE
Product synchronization issue

### ISSUE THREAD

**Issue by raphael0202** [openfoodfacts/open-prices#243]
This product (https://world.openfoodfacts.org/product/7792052075169/cereal-de-arroz-con-chocolate-rock-choc) was uploaded on Open Food Facts on 2024-02-26, but it's not synced in Open Prices database (as of 2024-03-06).

**Comment by raphodn** [comment]
the product in Open Prices : https://prices.openfoodfacts.org/app/products/7792052075169

and the JSON returned
```
{
    "code": "7792052075169",
    "id": 3081469,
    "source": null,
    "product_name": null,
    "product_quantity": null,
    "product_quantity_unit": null,
    "categories_tags": [],
    "brands": null,
    "brands_tags": [],
    "labels_tags": [],
    "image_url": null,
    "nutriscore_grade": null,
    "unique_scans_n": 0,
    "price_count": 1,
    "created": "2024-02-26T02:43:16.810574Z",
    "updated": null
}
```

**Comment by dq18** [comment]
Seems to me that:
https://github.com/openfoodfacts/open-prices/blob/21c27930250793092c9859de24430342626db59e/app/tasks.py#L140
is the blocking point. Newly created products in Open Prices not yet in OFF will have their source put to `null` by default

**Comment by raphodn** [comment]
It seems that the PR #250 did not fix the problem.

For example, today a price for `3384630300103` was added ([here](https://prices.openfoodfacts.org/app/products/3384630300103)), the product did not exist in OFF yet (added afterwards [here](https://fr.openfoodfacts.org/produit/3384630300103/petite-tomme-de-chevre-giralp)), and the API returns `source=off`

**Comment by raphodn** [comment]
fixed in #257
